src/mono/wasm/runtime/http.ts

```diff
@@ -21,7 +22,12 @@ export function http_wasm_abort_request(abort_controller: AbortController): void
 export function http_wasm_abort_response(res: ResponseExtension): void {
     res.__abort_controller.abort();
     if (res.__reader) {
-        res.__reader.cancel();
+        res.__reader.cancel().catch((err) => {
+            if (err && err.name !== "AbortError") {
+                Module.printErr("MONO_WASM: Error in http_wasm_abort_response: " + err);
```

Comment:
Shouldn't we throw in such case? The user wanted to cancel, but it probably failed on something different. I see it as catching `Exception` in .NET code when I want to catch just `OperationCancelledException`.

Reply:
The caller of cancel() is called from cancellation token registration, which probably doesn't propagate further ? Anyway, that only returns promise.
But the caller of `catch` here is a browser I think. I don't know who to propagate it to the user code.
Also, the user doesn't have corrective action probably.
Overall I think logging is fine until we know what are other ways how it could fail.

Reply:
I see, and we can't make `http_wasm_abort_response` async, because on the side, `HttpContent` doesn't implement `IAsyncDisposable`.

Okay than 👍 Thanks for the explanation.